### PR TITLE
Check if db is not empty

### LINKED ISSUE #2: Create new products if there are no products in db
There is no need to create new products if products already exist in the database. Please add the condition product-seeder.json to create new products ONLY if there are no products in the database.

### PR DESCRIPTION
**Pull request by lerabo**

### CODE REVIEW

**Comment by lerabo** [comment]
product-seeder.js is improved

File: seed/product-seeder.js [hunk 1]
```diff
@@ -2,9 +2,7 @@ const mongoose = require("mongoose");
 const Product = require("../models/product");
 const { faker } = require("@faker-js/faker");
 
-mongoose.connect(
-  "mongodb+srv://vercel-admin-user:QkFv0Usb0jQLTZjC@cluster0.43cr434.mongodb.net/shopping_cart?retryWrites=true&w=majority"
-);
+mongoose.connect("mongodb+srv://vercel-admin-user:QkFv0Usb0jQLTZjC@cluster0.43cr434.mongodb.net/shopping_cart?retryWrites=true&w=majority");
 
 const deleteData = () => {
```
Comment (VladBo) [inline seed/product-seeder.js]: You need to check here if products exist instead of deleting existing products.

File: seed/product-seeder.js [hunk 2]
```diff
@@ -17,30 +15,33 @@ const deleteData = () => {
...
+      if (done === products.length) {
+        mongoose.disconnect();
+      }
+    });
   }
 };
 
-seedData();
+
+const query = Product.find();
```
Comment (VladBo) [inline seed/product-seeder.js]: Remove product.deleteMany and replace by this query. Also please use async/await as all mongoose methods like find/save/delete are asynchronous. https://www.youtube.com/watch?v=V_Kr9OSfDeU

File: seed/product-seeder.js [hunk 3]
```diff
@@ -16,31 +16,37 @@ const deleteData = () => {
...
-        description: faker.commerce.productDescription(),
-        price: faker.commerce.price(10, 90, 0),
-      });
-      products.push(item);
-    }
-    console.log("seedData!");
+async function seedData() {
+  try {
+    const query = await Product.find();
+    query.count(function (err, count) {
```
Comment (VladBo) [inline seed/product-seeder.js]: You can use .length property here instead, check if it's not empty and there will be no need for query.count in the next line.

**Review by VladBo — CHANGES_REQUESTED** [review]

**Review by VladBo — APPROVED** [review]

**Pull request merged**